lua/telescope/builtin/internal.lua

```diff
@@ -403,27 +403,30 @@ internal.help_tags = function(opts)
 end
 
 internal.man_pages = function(opts)
-  local pages = utils.get_os_command_output(opts.man_cmd or { 'apropos', '--sections=1', '' })
+  opts.sections = utils.get_default(opts.sections, '1')
+  opts.man_cmd = utils.get_default(
+    opts.man_cmd,
+    vim.fn.has'mac' and {'apropos', ' '} or {'apropos', ''}
```

Comment:
`apropos ' '` and `apropos ''` is not the same. 
So we are probably looking for `apropos -w '*'` for both

Reply:
hmm... apropos in macOS is so primitive and has no options such as `-w`.

[apropos Man Page - macOS - SS64.com](https://ss64.com/osx/apropos.html)

I saw here and knew `apropos ' '` is enough for this time.

Reply:
Oh okay. I just did `apropos ' ' | wc -l` and `apropos '' | wc -l` and got a different count. 😆 Lets keep it that way